spark/src/test/java/gaffer/spark/generator/FilmGenerator.java

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
Instead of copying these files from the examples module, would it be possible to add a dependency on the example module in the pom and just reference them?